s3tests/generate_objects.py

```diff
@@ -43,7 +43,7 @@ def upload_objects(bucket, files, seed):
     name_generator = realistic.names(15, 4, seed=seed)
 
     for fp in files:
-        print >> sys.stderr, 'sending file with size %dB' % fp.size
+        sys.stderr.write('sending file with size %dB\n' % fp.size)
```

Comment:
Here and after for py2/py3 compatibility I guess you better use print from future package, like this:
```
from __future__ import print_function
.
.
.
print('sending file with size %dB' % fp.size, file=sys.stderr)
```